Gifski/util.swift

```diff
@@ -64,6 +64,55 @@ class SSView: NSView {
 }
 
 
+extension NSWindow {
+	static let defaultContentSize = CGSize(width: 480, height: 300)
+
+	static var defaultContentRect: CGRect = {
```

Comment:
@LarsJK Do you have any idea of how we can programmatically replicate the NIB window behavior of automatically stacking new windows not completely on top of each other?

<img width="817" alt="screen shot 2018-03-19 at 12 57 15" src="https://user-images.githubusercontent.com/170270/37580222-220fd83e-2b75-11e8-89fb-d0603a0fefaa.png">

Reply:
Not really.. I guess you would have to loop through `NSApplication.shared.windows`, or maybe you just need the keyWindow?
What is the standard cascading behaviour? Will it just add a new window at the same position as the keyWindow + the titles height?

Reply:
I don't really know. I've added a TODO in the code. I'll look into it at some later time.